packages/reporter/src/commands/commands.scss

```diff
@@ -377,6 +377,13 @@
     }
   }
 
+  .command-name-log {
+    .command-message {
+      white-space: initial;
+      word-wrap: break-word;
+    }
```

Comment:
Let's add `line-height: 1.5` to space out the lines a bit more.

Reply:
Done.